src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
> Would it be better to always force use of the setters by making the class vars private?
It would be, but this it would prevent the similar usage pattern to System.out and err (they're public static final, but you set via native setOut/Err methods). We can fully encapsulate, but to do that the API will look like this:

```
Std.out().println()
```

To completely mimick the Java runtime classes we'd likely have to go native or modify the JRE. I figured this would probably just be a stop-gap until a proper logging rework can be done. If the method access isn't too uncomfortable, I can fully-encapsulate.